spec/domain/operations/product_selection_effects/dchbx_product_selection_effects_spec.rb

```diff
@@ -910,7 +910,7 @@
   end
 
   let!(:prior_year_product) do
-    product = BenefitMarkets::Products::Product.all.by_year(2023).first
```

Comment:
@ATBull81 one thing, can we make the year dynamic? Something like `TimeKeeper.date_of_record.year`?

Reply:
https://github.com/ideacrew/enroll/pull/4887/commits/5acc396f5f5b13fd9a0f4bac9c7b5ade2bc5bc32 yep, there was ``previous_oe_year`` that already existed that basically did that job, started using that everywhere

Reply:
I was looking at the wrong thing. Looks good